analysers/trials/cpu.py

```diff
@@ -29,9 +32,26 @@
 
```

Comment:
You should apply caching and configuration passed from the spark-tasks-sender also to the analysers script working on trials, and in general to all the scripts.